modules/tip/logic/tipLogic.js

```diff
@@ -189,6 +194,9 @@ const TipLogic = {
         return includesRemoteUpdated || notIncludesRemote;
       });
 
+      // Send appropriate notifications for claims
+      await insertOrUpdateClaims.asyncMap(NotificationLogic.handleClaim);
```

Comment:
should probably be after claim with only the result that was actually inserted, to additionally avoid duplicates.